deploy/template.yaml

```diff
@@ -1883,7 +1755,7 @@ Resources:
     Type: AWS::Logs::SubscriptionFilter
     Condition: IsNotDevelopment
     Properties:
-      DestinationArn: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prod"
+      DestinationArn: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython"
```

Comment:
Can you example like I'm five what this is now doing?

Reply:
its going to a different subscription filter which is picked up by the cyber team - I think they use firehose / kinesis and a custom python tool to then capture and redirect to splunk indexes (no idea how the cyber side is setup) we are just shipping it off to their account to that destination